dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -161,29 +161,30 @@ HRESULT JustDoIt(PCWSTR path) noexcept try
         auto packageFullName{ std::wstring(line.c_str() + offset + 1) };
         THROW_HR_IF_MSG(E_UNEXPECTED, packageFullName.empty(), "line:%s", lineUtf8.c_str());
 
-        AddPackageIfNecessary(path, filename, packageFullName);
+        AddPackageIfNecessary(path, filename, packageFullName, forceDeployment);
     }
 
     return 0;
 }
 CATCH_RETURN();
 
-void AddPackageIfNecessary(PCWSTR path, const std::wstring& filename, const std::wstring& packageFullName)
+void AddPackageIfNecessary(PCWSTR path, const std::wstring& filename, const std::wstring& packageFullName, bool forceDeployment)
 {
     wprintf(L"Path: %s\n", path);
     wprintf(L"Filename: %s\n", filename.c_str());
     wprintf(L"PackageFullName: %s\n", packageFullName.c_str());
+    wprintf(L"forceDeployment:%d\n", forceDeployment);
 
     if (!NeedToRegisterPackage(packageFullName))
     {
         return;
     }
 
-    auto hr{ AddPackage(path, filename) };
+    auto hr{ AddPackage(path, filename, forceDeployment) };
     if (FAILED(hr))
     {
-        wprintf(L"AddPackage(): 0x%X Path:%ls Filename:%ls PackageFullName:%ls", hr, path, filename.c_str(), packageFullName.c_str());
-        THROW_HR_MSG(hr, "Path:%ls Filename:%ls PackageFullName:%ls", path, filename.c_str(), packageFullName.c_str());
+        wprintf(L"AddPackage(): 0x%X Path:%ls Filename:%ls PackageFullName:%ls forceDeployment:%d", hr, path, filename.c_str(), packageFullName.c_str(), forceDeployment);
+        THROW_HR_MSG(hr, "Path:%ls Filename:%ls PackageFullName:%ls forceDeployment:%d", path, filename.c_str(), packageFullName.c_str(), forceDeployment);
```

Comment:
%d = int == int32 = 4byte
bool == 1byte
(at least, the way our compiler implements these)

Suggestion: `...forceDeployment:%s", ..., forceDeployment ? "true" : "false");`